Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityDeviceManager.cs

```diff
@@ -366,23 +363,33 @@ public override void Enable()
             //NOTE: We update the source state data, in case an app wants to query it on source detected.
             for (var i = 0; i < numInteractionManagerStates; i++)
             {
-                var controller = GetController(interactionManagerStates[i].source);
-
-                if (controller != null)
-                {
-                    controller.UpdateController(interactionManagerStates[i]);
-                    inputSystem?.RaiseSourceDetected(controller.InputSource, controller);
-                }
+                CreateController(interactionManagerStates[i]);
             }
 
-            if ((inputSystem != null) &&
-                InputSystemProfile.GesturesProfile != null &&
+            if (InputSystemProfile.GesturesProfile != null &&
                 InputSystemProfile.GesturesProfile.WindowsGestureAutoStart == AutoStartBehavior.AutoStart)
             {
                 GestureRecognizerEnabled = true;
             }
         }
 
+        private void CreateController(InteractionSourceState interactionSourceState)
```

Comment:
Why is this call createcontroller?

Reply:
Maybe UpdateControllerForInteractionSourceState? or UpdateInteractionSourceController?

Reply:
would also prefer UpdateController or any variation of that - CreateController is misleading